eclipse-theia/theia
# allow a user to change default keybindings

#### What it does


- fix #6878: allow a user to change default keybindings

#### How to test


- change some editor default keybinding, like rename symbol
- check that default keybinding does not work anymore

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
